contracts/multisig-prover/src/encoding/rkyv/mod.rs

```diff
@@ -51,7 +54,20 @@ pub fn encode(
     Ok(HexBinary::from(bytes))
 }
 
-pub fn to_verifier_set(vs: &VerifierSet) -> Result<axelar_rkyv_encoding::types::VerifierSet> {
+pub fn payload_digest(
+    domain_separator: &Hash,
+    verifier_set: &VerifierSet,
+    payload: &Payload,
+) -> error_stack::Result<Hash, ContractError> {
+    Ok(axelar_rkyv_encoding::hash_payload(
+        &domain_separator,
+        &to_verifier_set(verifier_set)?,
+        &axelar_rkyv_encoding::types::Payload::try_from(payload)?,
+        axelar_rkyv_encoding::hasher::generic::Keccak256Hasher::default(),
+    ))
+}
+
```

Comment:
Aligning APi's with repo usage, making our rkyv module to match the signatures from other encoding modules for the sake of predictability.